### PR TITLE
Logout 

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:

* Add feature file for log out
* Add steps for step definition 

#### What I have learned working on this feature: 
* To use background in feature files
* To use warden
* To use destroy method

#### Screenshots:
skjermbilde 2018-03-16 kl 13 46 51

### CODE REVIEW

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 = link_to "Sign up", new_user_registration_path
 = link_to "Sign in", new_user_session_path
-
+= link_to "Log out", destroy_user_session_path, method: :delete
```
Comment (oliverochman) [inline app/views/home/index.html.haml:5]: Do you need to state the method here?
Does it work without it?

**Review by oliverochman — COMMENTED** [review]
Looks good to me!

File: app/views/home/index.html.haml (hunk 1 above)
Comment (isabellelii) [reply app/views/home/index.html.haml:5]: We added it because it didn't work without for some reason

**Review by isabellelii — APPROVED** [review]

**Review by davidboll — APPROVED** [review]
I think this look good

**Pull request merged**